src/project/internal/projectconfiguration.cpp

```diff
@@ -365,6 +368,16 @@ void ProjectConfiguration::setHomeScoresPageTabIndex(int index)
     m_homeScoresPageTabIndex = index;
 }
 
+IProjectConfiguration::HomeScoresPageViewType ProjectConfiguration::homeScoresPageViewType() const
+{
+    return settings()->value(HOME_SCORES_PAGE_VIEW_TYPE).toEnum<HomeScoresPageViewType>();
+}
+
+void ProjectConfiguration::setHomeScoresPageViewType(HomeScoresPageViewType type)
+{
+    settings()->setSharedValue(HOME_SCORES_PAGE_VIEW_TYPE, Val(type));
```

Comment:
Shouldn't it be setLocalValue? (considering [this comment ](https://github.com/musescore/MuseScore/pull/18853/files#diff-d5c3002ec488832c30d97940dae02163d3801ea467d62334da11c5e5002a958eR35))